pom.xml

```diff
@@ -222,7 +222,7 @@
 						<goals>
 							<goal>jar</goal>
 						</goals>
-						<phase>package</phase>
+						<phase>site</phase>
```

Comment:
It is not the javadocs that we are wanting to avoid generating, but rather the config props which is controlled [here](https://github.com/spring-projects-experimental/spring-grpc/blob/8d20f4a6d918579616d39cb834f12757d225b1bd/spring-grpc-docs/pom.xml#L109). 

We do not generate our docs until the `./mvnw -pl spring-grpc-docs antora` command is run ([docs.yml](https://github.com/spring-projects-experimental/spring-grpc/blob/8d20f4a6d918579616d39cb834f12757d225b1bd/.github/workflows/docs.yml#L38)). 

We would like to make this config props generator run at that time. 

If this were Gradle it would be super simple to create a custom task named "generateConfigProps" (or something like that). With Maven it is not as simple.